sdk/spring/tests.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+jobs:
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+        TimeoutInMinutes: 240
+        MaxParallel: 2
+        ServiceDirectory: spring
+        Matrix:
+          Linux - Java 8 (AzureCloud):
```

Comment:
@hemanttanwar @saragluna 
The Java team needs to comment on this but I think the test matrix needs to be more than just linux and Java 8. Similarly, should this be testing with more than just AZURE_TEST_HTTP_CLIENTS: netty?

Reply:
@JimSuplizio Yes, the matrix needs to be more than just Linux - Java 8 (just using only `Linux - Java 8` to save testing time). I will remove the matrix section in tests.yml in this PR to use the default matrix defined in the test template.